recipe/build.sh

```diff
@@ -23,6 +23,7 @@ cmake ${CMAKE_ARGS} -D CMAKE_INSTALL_PREFIX=$PREFIX \
       -D ZLIB_ROOT=$PREFIX \
       -D CURL_ROOT=$PREFIX \
       -D LAPACK_LIBRARIES=$PREFIX/lib/liblapack${SHLIB_EXT} \
+      -D GMT_ENABLE_OPENMP=TRUE \
```

Comment:
Is `GMT_USE_THREADS=TRUE` supposed to be set too? I see it mentioned in some places like https://docs.gmt-china.org/6.4/install/build-source-unix/#gmt and https://github.com/GenericMappingTools/gmt/pull/970

Reply:
Yes, `GMT_USE_THREADS` use GLib's GThreads component to parallel some codes.

Reply:
Ok, done at 7ccb1b88462d676cc22488865dd2a354078193f6

Reply:
Need to add libgdal (https://anaconda.org/conda-forge/libglib) and/or glib (https://anaconda.org/conda-forge/glib) as a dependency.